src/lxc/tools/arguments.h

```diff
@@ -160,6 +161,11 @@ struct lxc_arguments {
 #define OPT_VERSION OPT_USAGE - 1
 #define OPT_RCFILE OPT_USAGE - 2
 
+#define OPT_SHARE_NET OPT_USAGE + 1
```

Comment:
(not sure) if these are now in a shared file, should these become OPT_USAGE -3 etc , so that OPT_USAGE + 1 is always usable by a shared file?

Reply:
Happy to do it either way, just let me know.

Reply:
Yes, please change it.